Doc/library/dataclasses.rst

```diff
@@ -292,6 +292,10 @@ Module contents
 
     .. versionadded:: 3.10
 
+   - ``doc``: optional docstring for this field.
```

Comment:
I think we should mention that this is only used for `__slots__`. To a casually user I think it would be a surprise to know that without `__slots__` this is completely ignored.

Reply:
Done!